components/frontend/src/measurement/MeasurementValue.js

```diff
@@ -1,39 +1,91 @@
+import "./MeasurementValue.css"
+
 import { bool, string } from "prop-types"
 import { useContext } from "react"
-import { Icon } from "semantic-ui-react"
 
 import { DataModel } from "../context/DataModel"
-import { Label, Popup } from "../semantic_ui_react_wrappers"
-import { datePropType, metricPropType } from "../sharedPropTypes"
+import { Icon, Label, Message, Popup } from "../semantic_ui_react_wrappers"
+import { datePropType, measurementPropType, metricPropType } from "../sharedPropTypes"
+import { SOURCE_ENTITY_STATUS_NAME } from "../source/source_entity_status"
 import {
     formatMetricValue,
     getMetricScale,
+    getMetricUnit,
     getMetricValue,
     isMeasurementRequested,
     MILLISECONDS_PER_HOUR,
+    sum,
 } from "../utils"
 import { TimeAgoWithDate } from "../widgets/TimeAgoWithDate"
 import { WarningMessage } from "../widgets/WarningMessage"
 
-function measurementValueLabel(stale, updating, value) {
+function measurementValueLabel(hasIgnoredEntities, stale, updating, value) {
+    const ignoredEntitiesIcon = hasIgnoredEntities ? <Icon name="hide" /> : null
     if (stale) {
-        return <Label color="red">{value}</Label>
+        return (
+            <Label color="red">
+                {ignoredEntitiesIcon} {value}
+            </Label>
+        )
     }
     if (updating) {
         return (
             <Label color="yellow">
-                <Icon loading name="hourglass" /> {value}
+                <Icon loading name="hourglass" /> {ignoredEntitiesIcon} {value}
             </Label>
         )
     }
-    return <span>{value}</span>
+    return (
+        <span>
+            {ignoredEntitiesIcon}
+            {value}
+        </span>
+    )
 }
 measurementValueLabel.propTypes = {
-    stale: bool,
+    hasIgnoredEntities: bool,
     updating: bool,
+    stale: bool,
     value: string,
 }
 
+function ignoredEntitiesCount(measurement) {
+    const count = { fixed: 0, false_positive: 0, wont_fix: 0 }
+    measurement.sources?.forEach((source) => {
+        Object.values(source.entity_user_data ?? {}).forEach((entity) => {
+            if (Object.keys(count).includes(entity.status)) {
+                count[entity.status]++
+            }
+        })
+    })
+    return count
+}
+ignoredEntitiesCount.propTypes = {
+    measurement: measurementPropType,
+}
+
+function ignoredEntitiesMessage(measurement, unit) {
+    const count = ignoredEntitiesCount(measurement)
+    let summary = `The measurement value excludes ${sum(count)} ${unit}.`
+    let details = ""
+    ;["false_positive", "fixed", "wont_fix"].forEach((status) => {
```

Comment:
What's this weird semicolon shorthand? Is it not a better idea to turn this list of three options into a constant which can also be used elsewhere (i.e. when assigning the value 0 to each of them in a dict)?